### ISSUE TITLE
fstab parameter of mount module won't work

### ISSUE THREAD

**Issue by 64er** [ansible/ansible-modules-core#3153]
##### Issue Type:
- Bug Report
##### Ansible Version:

ansible 2.0.0.2
  config file = /etc/ansible/ansible.cfg
  configured module search path = /usr/share/ansible
##### Environment:

Ubuntu 14.04 64Bit
##### Summary:

set fstab=**/tmp/fstab** in mount module result in 'can't find mount point in **/etc/fstab**'
##### Steps To Reproduce:

```
./test-module -m ../lib/ansible/modules/core/system/mount.py -a "src=/dev/sda9 name=/tmp/mnt fstype=ext3 state=mounted **fstab=/tmp/fstab**"
```
##### Expected Results:

```
changed: ok
```

and a line in /tmp/fstab and 'mount -a' should show the mounted device
##### Actual Results:

```
***********************************
PARSED OUTPUT
{
    "failed": true, 
    "invocation": {
        "module_args": {
            "dump": null, 
            "fstab": **"/tmp/fstab"**, 
            "fstype": "ext3", 
            "name": "/tmp/mnt", 
            "opts": null, 
            "passno": null, 
            "src": "/dev/sda9", 
            "state": "mounted"
        }
    }, 
    "msg": "Error mounting /tmp/mnt: mount: /tmp/mnt konnte nicht in **/etc/fstab oder /etc/mtab** gefunden werden\n"
}
```

**Comment by mscherer** [comment]
Yep, it doesn't work on Linux, it seems to be only on freebsd for the current code. There is the option -T for Linux, could you give a try to the commit I propose ?

**Comment by 64er** [comment]
I tried your commit. I got an error like "no Option -T for mount". I looked into the manpage of mount and couldn't find -T on my Ubuntu 14.04 system.

**Comment by mscherer** [comment]
Ok so that's only for newer coreutils. I guess we can add that to the doc and maybe some code to verify that, but I do not see any good way.

**Comment by alikins** [comment]
Looks like the fstab option is only applied on freebsd, so needs docs updated.

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by jesse-welch** [comment]
This change appears to break mount tasks on RHEL6 / CentOS6.

Error mounting < /Some/Path > : /bin/mount: invalid option -- 'T'

**Comment by mscherer** [comment]
Indeed, can you open a new issue, I will work on a fix.

**Comment by jesse-welch** [comment]
@mscherer  Thanks, I'll open a new issue today with the necessary details.  From my initial review, it appears that the -T option was introduced somewhere between:
'mount from util-linux-ng 2.17.2' 
and 
'mount from util-linux 2.23.2'.
